packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/guides/telecom-telephony-guides.constants.js

```diff
@@ -0,0 +1,111 @@
+export const GUIDES = {
```

Comment:
moved from Angular.js to ES6 constants (from `packages/manager/apps/telecom/src/app/telecom/telephony/telecom-telephony.constant.js`)